release notes/v0.39.0.md

```diff
@@ -0,0 +1,60 @@
+
+As it's hopefully clear from the name, our usual guarantees of API stability won't apply to these modules while they are still experimental. We reserve the right to make breaking changes in `experimental` modules, up to and including completely droping them. We don't expect big breaking changes will need to happen often, but we want to be clear they are not impossible. Finally, whenever we feel like a new experimental API has been stabilized, we will move it to its final stable import path, but there will be at least one k6 release where the module will be accessible through both import paths, so its users have time to move on to the new stable import path.
+
+## Native support for ECMAScript modules
+
+At the moment, k6 has support for [ECMAScript modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules) (ESM, i.e. `import`, `export`, etc.) via [automatic transpilation](https://k6.io/docs/using-k6/javascript-compatibility-mode/) of scripts by the built-in [Babel.js](https://babeljs.io/). That mostly works, but it has caused some performance and compatibility problems ([#824](https://github.com/grafana/k6/issues/824#issuecomment-915918351) and [#2168](https://github.com/grafana/k6/issues/2168#issuecomment-1014276690) among others), so we want to support ESM modules and all other ES6 features directly in k6, without the need for Babel.js ([#2296](https://github.com/grafana/k6/issues/2296#issuecomment-993356717)). [goja](https://github.com/dop251/goja), the JavaScript runtime we use to elavulate k6 scripts, doesn't yet have native ESM support, so we are currently working on adding it there to then be able to support ECMAScript modules natively in k6!
+
+That work has been ongoing for a while and we're making progress, but it will likely not be ready in time for the next k6 v0.40.0 release. We are mentioning it here because we will probably need to make a few minor breaking changes and fixes of currently undefined behavior in k6 to support ESM modules natively. For example, at the moment, some values like the consolidated script `options` were unintentionally made available globally, in all JS modules of a test, instead of just in the exported `options` value from the main JS module. That is not the intended or documented behavior, it's somewhere between a bug and undefined behavior, and we'll need to fix it ([#2571](https://github.com/grafana/k6/pull/2571)) and other similar issues like it, starting in k6 v0.40.0.
+
+We don't expect many scripts to break because of these fixes, but we'll nevertheless announce them as breaking changes in the release notes of the k6 version that they happen in.
```

Comment:
```suggestion
We don't expect many scripts to break because of these fixes, but we'll nevertheless announce them as breaking changes in the `v0.40.0` release notes of the k6 version that they happen in.
```

Reply:
hm the sentence doesn't make sense with this addition :confused: